.storybook/main.ts

```diff
@@ -79,9 +79,9 @@ const config: StorybookConfig = {
     });
   },
 
-  docs: {
-    autodocs: true,
-  },
+  // docs: {
+  //   autodocs: true,
+  // },
```

Comment:
Why was this commented out?

Reply:
I'm not sure we are using this feature and having it or not having it doesn't seem to change anything.